service/src/main/kotlin/app/cash/backfila/client/BackfilaClientServiceClientProvider.kt

```diff
@@ -1,25 +1,28 @@
 package app.cash.backfila.client
 
 import com.google.inject.Provides
+import misk.client.HttpClientConfig
 import java.time.Duration
 import javax.inject.Named
 import misk.client.HttpClientEndpointConfig
+import misk.client.HttpClientsConfig
 import misk.inject.KAbstractModule
 
-val DEFAULT_HTTP_CLIENT_ENDPOINT_CONFIG = HttpClientEndpointConfig(
-    // Allow RunBatch requests to take a long time.
-    readTimeout = Duration.ofSeconds(30L),
-    // Use a large number, we limit parallel calls ourselves.
-    maxRequests = 1000,
-    maxRequestsPerHost = 1000
+val DEFAULT_HTTP_CLIENTS_CONFIG = HttpClientsConfig(
+  hostConfigs = linkedMapOf(
+    ".*" to HttpClientConfig(
```

Comment:
huh, is regex in the key a new feature?

Reply:
The idea is that backfila is no longer in the middle. Its how you define what urls get what config.

Reply:
Yeah, we tried to make routing of requests abstracted away. For this to work, I changed the HttpClientsConfig to be URL based. This way, Skim can define default routing settings (like `*.gns.square` always going through the abstract socket), and clients can just provide an URL and delegate the correct routing of this URL to Misk/Skim/OS.